bitnami/dataplatform-bp1/values.yaml

```
@@ -170,3 +261,110 @@ solr:
     ##
     servers:
       - '{{ .Release.Name }}-zookeeper'
+
+## Wavefront Subchart parameters
+##
+wavefront:
+  enabled: false
+  ## This is a unique name for the cluster (required)
+  ## All metrics will receive a `cluster` tag with this value
+  ##
+  clusterName: KUBERNETES_CLUSTER_NAME
+  ## Wavefront URL (cluster) and API Token (required)
+  ##
+  wavefront:
+    url: https://YOUR_CLUSTER.wavefront.com
+    token: YOUR_API_TOKEN
+    ## Name of an existing secret containing the token
+    ##
+    existingSecret:
+  ## Wavefront Collector is responsible to get all Kubernetes metrics from your cluster.
+  ## It will capture Kubernetes resources metrics available from the kubelets,
+  ## as well as auto-discovery capabilities.
+  ##
+  collector:
+  ## Rules based discovery configuration
+  ## Ref: https://github.com/wavefrontHQ/wavefront-kubernetes-collector/blob/master/docs/discovery.md
+  ##
```

Could you fix the indentation for this comment?

```suggestion
    ## Rules based discovery configuration
    ## Ref: https://github.com/wavefrontHQ/wavefront-kubernetes-collector/blob/master/docs/discovery.md
    ##
```